pkg/policies/branch/branch.go

```diff
@@ -566,6 +582,10 @@ func fix(ctx context.Context, rep repositories, c *github.Client,
 				pr.RequiredPullRequestReviews.RequiredApprovingReviewCount = mc.ApprovalCount
 				update = true
 			}
+			if mc.RequireCodeOwnerReviews && !pr.RequiredPullRequestReviews.RequireCodeOwnerReviews {
```

Comment:
Should L576 above also change to `if mc.RequireApproval || mc.RequireCodeOwnerReviews `?

Reply:
Also L567

Reply:
I've updated the help, as I think it doesn't make sense to use requireCodeOwnerReviews without requireApproval